packages/client/src/components/Modals/SavedSearchPanel.vue

```diff
@@ -142,6 +143,14 @@ export default {
       const criteriaObj = JSON.parse(criteria);
       return formatFilterDisplay(criteriaObj);
     },
+    notifyDeleted() {
+      this.$bvToast.toast('Search deleted', {
+        title: 'Success',
+        variant: 'success',
+        solid: true,
+        autoHideDelay: 2500,
+      });
+    },
```

Comment:
I didn't see any other delete patterns (I saw that users for example doesn't show anything), so I added a quick & dirty toast notification.